### ISSUE TITLE
Dark mode

### ISSUE THREAD

**Issue by Neustradamus** [ovh/manager#9836]
### Have you already contact our help centre?

- [X] Yes, I have contact the help centre.

### Describe the problem

Dear @ovh team,

We are in 2023, and currently there is no dark mode for the manager.

Can you add it?

Thanks in advance.

### Describe the solution

Add the dark mode for Manager.

### Describe alternatives you've considered

Change of company.

**Comment by Owen2k6** [comment]
i upvote this

**Comment by prplwtf** [comment]
why isn't this already a thing yet

**Comment by Gecko64** [comment]
Yes really, that kill my eyes each time i must change something in the manager.

**Comment by chozeur** [comment]
+1

**Comment by thomas-berrio** [comment]
+1

**Comment by Moose-08** [comment]
# PLEASE add this, I get flash banged with everything I click being on different pages

**Comment by Neustradamus** [comment]
@Denis-hamon: Can you look this missing feature?

**Comment by lunaluvmilo** [comment]
+1

**Comment by knonode** [comment]
+1000

**Comment by Denis-hamon** [comment]
I m redirecting this request to our 'General Customer Journey' team ;)

**Comment by cadegorawrz** [comment]
+10000000

**Comment by greguz** [comment]
Pretty please <3

**Comment by Neustradamus** [comment]
Happy New Year 2026 to all!

**Comment by Qbsoon** [comment]
Another year, still waiting

**Comment by abdofallah** [comment]
**heres a tamper monkey script for dark mode:** https://gist.github.com/abdofallah/c96d196a16024ac9af340ca18b75d356

---

gets the job done of reducing major eyes stress but still many elements have hard coded styles so they were skipped as they are rarely used/seen.

appreciate contributions for the pages you may use but is not handled fully by the script

**Comment by CyberMist2** [comment]
+1

**Comment by domi41** [comment]
For everyone (like me) that **feel actual pain** when they have to use the validist UI of OVH, I've found this alternative :

https://www.accessibilitycloud.com/

> This is not about comfort ; it's about pain.  Give your money to people who care.

**Comment by ErMaqui** [comment]
3 years after opening and still no info from dev team about this...?

**Comment by Neustradamus** [comment]
It is like the Cookies manager problem too...
- https://github.com/ovh/manager/issues/22970

All "+1" are needed on this second ticket too...

**Comment by yfdida** [comment]
No worry folks, Dark mode is coming ;) it's planned you will see it popping very soon
